apps/api/src/modules/summary/summary.service.ts

```diff
@@ -31,9 +32,24 @@ export class SummaryService extends CoreService<Summary> {
     if (inputFile) {
       const { createReadStream, filename } = await inputFile;
 
-      const uniqueIdentifier = uuidv4();
+      const uniqueIdentifier = uuidv4().replace(/-/g, '').substring(0, 10);
       const modifiedFilename = `${uniqueIdentifier}_${filename}`;
-      const fileLocation = join(process.cwd(), `./src/upload/${modifiedFilename}`);
+      const uploadPath = process.env.Upload_Path
+        ? process.env.Upload_Path.replace(/[^\w\s/]/g, '')
+        : null;
+
+      if (uploadPath) {
+        const absoluteUploadPath = resolve(uploadPath);
+        await fs.ensureDir(absoluteUploadPath);
+      } else {
+        const defaultPath = join(process.cwd(), 'src', 'uploads');
+        await fs.ensureDir(defaultPath);
+      }
+
+      const fileLocation = join(
+        uploadPath || resolve(process.cwd(), 'src', 'uploads'),
```

Comment:
Keep the path value as a constant and reuse it.
We should just have one upload path.
Also make sure you are creating the folder in  main.ts along with log folder first.
Its good to ensureDir as a rare case when someone deletes the folder manually.

Reply:
Done